HiddenWallet.Daemon/Tor.cs

```diff
@@ -26,6 +26,8 @@ public static class Tor
 		{
 			var ctsToken = CircuitEstabilishingJobCancel.Token;
 			State = TorState.NotStarted;
+			TryBroadcast();
```

Comment:
Instead of manually putting `TryBroadcast()` every time the state is changed, could you rather broadcast in a setter function? Something like this:
```cs
private TorState _state;
public TorState State
{
	get { return _state; }
	set
	{
		if (value != _state)
		{
			_state = value;
			try{TryBroadcast();}catch{}
		}
	}
}
```